$OpenBSD$

archivers/minizip is too new to be used, force using the older bundle
https://github.com/desktop-app/cmake_helpers/pull/156

Index: cmake/external/minizip/CMakeLists.txt
--- cmake/external/minizip/CMakeLists.txt.orig
+++ cmake/external/minizip/CMakeLists.txt
@@ -7,7 +7,7 @@
 add_library(external_minizip INTERFACE IMPORTED GLOBAL)
 add_library(desktop-app::external_minizip ALIAS external_minizip)
 
-if (DESKTOP_APP_USE_PACKAGED)
+if (DESKTOP_APP_USE_PACKAGED AND DESKTOP_APP_USE_PACKAGED_MINIZIP)
     find_package(PkgConfig REQUIRED)
     pkg_check_modules(MINIZIP IMPORTED_TARGET minizip)
 
